src/index.js

```diff
@@ -179,10 +180,9 @@ const compare = (condition, value, attributes) => {
       return Array.isArray(value) && Array.isArray(compareValue) && value.every(x => compareValue.includes(x));
 
     case 'matches':
-      if (compareValue === undefined) return undefined;
```

Comment:
had to take this line out because the schema validates this is defined.   there would not be 100 percent coverage without.

Reply:
`compareValue` here is the value from the attributes, not the `value` from the comparison. To hit this path you'll just need where the attribute being referenced doesn't exist.